x-pack/plugins/uptime/public/components/monitor/synthetics/script_expanded_row.tsx

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiLoadingSpinner } from '@elastic/eui';
+import React, { useCallback, useEffect, FC } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { Ping } from '../../../../common/runtime_types';
+import { getJourneySteps, getStepScreenshot } from '../../../state/actions/journey';
+import { JourneyState } from '../../../state/reducers/journey';
+import { journeySelector } from '../../../state/selectors';
+import { EmptyStepState } from './empty_journey';
+import { ExecutedJourney } from './executed_journey';
+import { ConsoleOutputStepList } from './console_output_steps';
+
+interface ScriptExpandedRowProps {
+  checkGroup?: string;
+}
+
+export const ScriptExpandedRow: React.FC<ScriptExpandedRowProps> = ({ checkGroup }) => {
```

Comment:
One state we haven't handled, that I did observe, is where the steps all succeed, but something causes the script to fail. We now detect this in heartbeat by sending an error document if no `journey/end` event is released. In this case we'll get an error like: 

```json
{
          "error" : {
            "message" : "journey did not finish executing, 3 steps ran",
            "type" : "io"
          },
}
```

I think we should put something in, maybe a callout displaying the most recent error at the top of the expanded list.

![image](https://user-images.githubusercontent.com/131427/94632218-a662be00-028f-11eb-92e7-0f3228dd254e.png)

Reply:
We should chat about this - is there a `synthetics.type` I should watch for and select the most recent to supply the content for an error callout? I see the illustration of a supposedly `down` journey with no `failed` steps here and agree it should be addressed.